lambda_functions/crud/operations.py

```diff
@@ -295,3 +331,43 @@ def get_all_review_questions_db():
     review_questions = session.query(ReviewQuestion).all()
     return review_questions
 
+
+def get_open_item_for_user_db(user):
+    """Inserts a new review answer into the database
+
+    Parameters
+    ----------
+    user: User
+        The user that should receive a case
+
+    Returns
+    ------
+    item: Item
+        The case to be assigned to the user
+    """
+
+    session = get_db_session()
+
+    review_type = 'needs_junior'
+    
+    # Senior detectives can get level 1 or level 2 reviews
+    if user.level < 1:
+        review_types = ['needs_junior', 'needs_senior']
+        review_type = random.choice(review_types)
+
+    sql_query = """SELECT items.id, min(submissions.submission_date) as oldest_submission,
```

Comment:
Would this be more simple using ORM?

Reply:
I tried that but found that it would be a huge hassle to implement this logic with ORM, especially the filtering by the minimum submission_date.